### DISCUSSION TITLE
Templater date is always wrong in Australia

### DISCUSSION THREAD

**Discussion by paulsnedden** [SilentVoid13/Templater#754] [Help]
I have just used the following in a Templater template I built
`(<% tp.date.now("YYYY-MM-DD") %>)`

The date that it inserted is 2021-12-01, but "today" in Australia right now is 2021-12-02 (10pm).

Is there a timezone setting or something I'm missing that may fix this, as it's extremely annoying to constantly have the wrong date in note titles and what not.

**Comment by paulsnedden** [comment]
Using `{{date:YYYY-MM-DD}}`, obviously outside of Templater, yields the correct date.

**Comment by AB1908** [comment]
Can you try printing a full date time to see which time zone is being used?

**Comment by shabegom** [comment]
I would expect it to be using your computer's set time zone. 

As a workaround you could try using the underlying moment js library and setting the utc offset:

`<% moment().utc(10).format('YYYY-MM-DD') %>`

I don't know where in Australia you are, so the utc but might need some adjusting.
